lib/kb_hmmer/kb_hmmerImpl.py

```diff
@@ -1828,11 +1852,12 @@ def HMMER_Local_MSA_Group_Search(self, ctx, params):
             if not os.path.exists(hmmer_dir):
                 os.makedirs(hmmer_dir)
 
-            if msa_type_name == 'MSA':
+            if msa_type_name != 'MSA':
+                raise ValueError('Cannot yet handle input_msa type of: '+msa_type_name)
+            else:
```

Comment:
You don't need an else here because the exception will bail out already

Reply:
I did this so the very short failure case would be much easier to read than at the end of the logical block (and more importantly I wouldn't have to re-indent a huge block which would look like non-real deltas).  I know it's unecessary, but it doesn't really cost anything.  OK?

Reply:
Please support my laziness :-)

Reply:
of course that argument based on line deltas is out the window now that I've standardized whitespace using autopep8, leaving only laziness (and a desire to not mess up the existing nesting) as my defense

Reply:
Let me clarify, placing the `if msa_type_name != 'MSA':` check at the top of the logic is a very good idea. I was just pointing out (for future reference) that because you are `raise`-ing an exception in the block you don't need the `else:` and indent. That code will never be executed if the exception is raised. You needn't change the code here but avoiding unneeded indents helps keep your lines short and readable.